### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by omgovich**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,21 @@
...
 
-Similar packages suggestions for the NPM ecosystem
+<div align="center">
+  Similar packages suggestions for the NPM ecosystem.<br />
+  <a href="https://bundlephobia.com/">Bundlephobia</a> team project
+</div>
+
+## How to add a package or category
+
+The best way to help us improve our registry is to create a Pull Request updating `src/similar-npm-packages.ts` file.
```
Comment (pastelsky) [inline README.md]: Currently folks can just submit an issue, and I do a monthly round-up of tickets and add those as suggestions. Creating a PR is appreciated, but optional.

One of the reasons for this is that the additions of `tags` is a manual process that cannot be easily automated (I've had to manually tweak the weights of tags every time a new category is created).

Do you think folks that don't have time to raise a PR should be able to submit issues?

Comment (omgovich) [reply README.md]: I would prefer to receive most of suggestions via PRs, but yeah, submitting an issue is also should be available.
Let's create ISSUE_TEMPLATE (the same as PR one) then and tweak the guide a bit.

Comment (omgovich) [reply README.md]: Done

File: README.md [hunk 2]
```diff
@@ -1,6 +1,19 @@
...
+<div align="center">
+  Similar packages suggestions for the NPM ecosystem.<br />
+  <a href="https://bundlephobia.com/">Bundlephobia</a> team project
+</div>
+
+## How to add a package or category
+
+If you want to help us improve the registry you could [create a new issue](https://github.com/pastelsky/similar-npm-packages/issues/new) describing a suggested package/category or change `src/similar-npm-packages.ts` file yourself with a pull request.
+
+Qualitative over quantitative. Before contributing you should know that we keep suggestions limited to 5 packages in all categories. If you're going to add a package to a "crowded" category, be ready to explain your suggestion. Qualitative over quantitative.
```
Comment (pastelsky) [inline README.md:17]: "Qualitative over quantitative" is repeated twice :)

Comment (omgovich) [reply README.md:17]: My bad)

**Review by pastelsky — APPROVED** [review]

**Pull request merged**